src/main/js/command.test.js

```diff
@@ -0,0 +1,101 @@
+jest.mock('child_process');
+jest.mock('get-installed-path');
+
+const { getInstalledPath } = require('get-installed-path');
+const { exec } = require('child_process');
+
+const {
+  initResult,
+  buildArguments,
+  parseResponse,
+  executeCommand,
+} = require('./command');
+
+const { ExecutablePath } = require('./constants');
+
+describe('#initResult', () => {
+  it('initializes result object', () => {
```

Comment:
we usually use `ŧest`, `it` is just an alias to `test` so it doesn't really matter